README.md

```diff
@@ -1 +1,172 @@
-# Module-1-Capstone
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🚀 Loom Video](#loom-video)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Module 1 Capstone Project] <a name="about-project"></a>
+
+>
+
+**[Capstone Project]** is a project made at the end of the first module at Microverse. It demonstrates what I learned in the first module.
+
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom Video <a name="loom-video"></a>
+
+> Link to the loom video of the project description.
+
+- [Loom Video Link](https://www.loom.com/share/5439f68a9d7a43fdaea4c2eee54b1c8f)
```

Comment:
- [ ] Let's keep the presentation video's duration shorter than 5 minutes as specified by the requirements. Cheers 👍